superset/tasks/cache.py

```diff
@@ -72,15 +73,15 @@ def get_form_data(
     return form_data
 
 
-def get_url(chart: Slice, extra_filters: Optional[Dict[str, Any]] = None) -> str:
+def get_url(chart: Slice) -> str:
     """Return external URL for warming up a given chart/table cache."""
     with app.test_request_context():
         baseurl = (
             "{SUPERSET_WEBSERVER_PROTOCOL}://"
             "{SUPERSET_WEBSERVER_ADDRESS}:"
             "{SUPERSET_WEBSERVER_PORT}".format(**app.config)
         )
-        return f"{baseurl}{chart.get_explore_url(overrides=extra_filters)}"
+        return f"{baseurl}/superset/warm_up_cache/?slice_id={chart.id}"
```

Comment:
this change will cause potential feature regression. This endpoint, get_url, used to accept extra_filters, so that it will hit slice with extra filters (like user open slice from Dashboard).

After your change, get_url can only send query based on saved params from slice, but user that already used this API will send wrong requests (since you dropped extra_filters)